keras/backend/cntk_backend.py

```diff
@@ -1038,7 +1038,22 @@ def resize_images(X, height_factor, width_factor, data_format):
         output = repeat_elements(output, width_factor, axis=2)
         return output
     else:
-        raise ValueError('CNTK Backend: Invalid dim_ordering:', data_format)
+        raise ValueError('CNTK Backend: Invalid data_format:', data_format)
+
+
+def resize_volumes(X, depth_factor, height_factor, width_factor, data_format):
```

Comment:
Please avoid capital letter variables (`X`) and add a docstring (same as the TF backend should be fine).